src/pages/FirstSearch.tsx

```diff
@@ -28,7 +29,9 @@ const FirstSearch: React.FunctionComponent<{}> = () => {
                 to={{
                   pathname: "/sheet",
                   state: {
-                    citizenid
+                    citizenid: SupportToken.is(citizenid)
+                      ? citizenid
+                      : citizenid.toUpperCase()
```

Comment:
Can we use a helper function to do this operation to avoid code duplication?

Reply:
you right, done.